lms/envs/common.py

```diff
@@ -2562,6 +2563,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # Management of per-user schedules
     'openedx.core.djangoapps.schedules',
     'rest_framework_jwt',
+
+    # Monitoring utilities for LMS
+    'lms.djangoapps.monitoring.apps.MonitoringConfig'
```

Comment:
Nit: Ideally, this new app would only be registered for the edX.org instance and not impact Open edX instances. This can be done by making it a Django App Plugin (that is still within edx-platform) and added in `setup.py`.  I'm keeping this as a Nit though since I don't believe the Plugin API currently allows adding plugins to the `MIDDLEWARE` list.

Reply:
This middleware basically is a no-op if the accompanying Django Setting isn't set, so I think it is fine as-is.